src/data/sparse_page_writer.h

```diff
@@ -6,8 +6,7 @@
  *   use in external memory computation
  * \author Tianqi Chen
  */
-#ifndef XGBOOST_DATA_SPARSE_BATCH_PAGE_H_
-#define XGBOOST_DATA_SPARSE_BATCH_PAGE_H_
+#pragma once
```

Comment:
Let's stick to the current convention of using include guards. Include guards with `#ifndef / #define` are supported by any standard-conforming C++ compilers, whereas `#pragma once` is not actually part of the C++ standard.